crates/nu-command/src/conversions/into/binary.rs

```diff
@@ -25,13 +35,15 @@ impl Command for SubCommand {
                 (Type::Bool, Type::Binary),
                 (Type::Filesize, Type::Binary),
                 (Type::Date, Type::Binary),
+                (Type::String, Type::String),
             ])
             .allow_variants_without_examples(true) // TODO: supply exhaustive examples
             .rest(
                 "rest",
                 SyntaxShape::CellPath,
                 "for a data structure input, convert data at the given cell paths",
             )
+            .switch("raw", "Convert to raw", Some('r'))
```

Comment:
as this only applies to `string`s, maybe we should mention it in the description of the option? e.g.
> Convert strings to raw binary. Do not apply on other datatypes.

Reply:
thanks!